src/rplidar_node.cpp

```diff
@@ -53,6 +53,7 @@ rplidar_node::rplidar_node(const rclcpp::NodeOptions & options)
   flip_x_axis_ = this->declare_parameter("flip_x_axis", false);
   scan_mode_ = this->declare_parameter("scan_mode", std::string());
   topic_name_ = this->declare_parameter("topic_name", std::string("scan"));
+  auto_standby_ = this->declare_parameter("auto_standby", true);
```

Comment:
@v-kiniv This looks good / useful overall, but I would appreciate it if this would not default to `true`. I don't want to break existing behavior for current users.

Reply:
No problem, I've set it to false by default